source/uwp/AdaptiveCards.sln

```diff
@@ -36,20 +36,20 @@ Global
 		{CF169157-F487-420C-8B9B-CA3ABE2BB209}.Release|x64.Build.0 = Release|x64
 		{CF169157-F487-420C-8B9B-CA3ABE2BB209}.Release|x86.ActiveCfg = Release|Win32
 		{CF169157-F487-420C-8B9B-CA3ABE2BB209}.Release|x86.Build.0 = Release|Win32
-		{B5DECA6E-D9EA-476D-9B7E-34D3B3044CD8}.Debug|ARM.ActiveCfg = Debug|ARM
```

Comment:
adding changes to AdaptiveCards.sln always leads to problems. unless it was intended, please revert it.